src/main/resources/assets/jei/lang/de_de.lang

```diff
@@ -1,45 +1,45 @@
 # Tooltips
 jei.tooltip.config=JEI-Konfiguration
-jei.tooltip.show.recipes=Zeige Rezepte
+jei.tooltip.show.recipes=Zeige die Rezepte
 jei.tooltip.show.all.recipes=Zeige alle Rezepte
-jei.tooltip.delete.item=Klicke zum Löschen
+jei.tooltip.delete.item=Klicke, um zu Löschen
 jei.tooltip.liquid.amount.with.capacity=%,d / %,d mB
 jei.tooltip.liquid.amount=%,d mB
-jei.tooltip.transfer=Bewege Items
+jei.tooltip.transfer=Bewege die Items
 jei.tooltip.recipe.ore.dict=Akzeptiert: %s
 jei.tooltip.item.colors=Farben: %s
 jei.tooltip.shapeless.recipe=Formloses Rezept
 jei.tooltip.cheat.mode.button.enabled=Cheat-Modus
-jei.tooltip.cheat.mode.how.to.disable.hotkey=Press %s to toggle it.
-jei.tooltip.cheat.mode.how.to.disable.no.hotkey=%s-click here to toggle it.
+jei.tooltip.cheat.mode.how.to.disable.hotkey=Klicke auf %s um es umzuschalten.
+jei.tooltip.cheat.mode.how.to.disable.no.hotkey=%s-klicke hier, um es umzuschalten.
 jei.tooltip.recipe.by=Recipe By: %s
 jei.tooltip.recipe.id=Recipe ID: %s
-jei.tooltip.not.enough.space=There is not enough space to display the ingredient list here.
-jei.tooltip.ingredient.list.disabled=The JEI overlay is disabled.
-jei.tooltip.ingredient.list.disabled.how.to.fix=Press %s to enable it.
+jei.tooltip.not.enough.space=Da ist nicht genug Platz, um die Zutaten-Liste hier anzeigen zukönnen.
+jei.tooltip.ingredient.list.disabled=Die JEI-Oberfläche ist deaktiviert.
+jei.tooltip.ingredient.list.disabled.how.to.fix=Klicke auf %s um es zu aktivieren.
 
 # Error Tooltips
 jei.tooltip.error.recipe.transfer.missing=Fehlende Items
 jei.tooltip.error.recipe.transfer.inventory.full=Inventar ist zu voll
-jei.tooltip.error.recipe.transfer.unknown=Unbekannter Fehler, siehe Logs
+jei.tooltip.error.recipe.transfer.unknown=Unbekannter Fehler, siehe Protokoll(Logs)
 jei.tooltip.error.recipe.transfer.no.server=Auf dem Server muss JEI installiert sein
 jei.tooltip.error.recipe.transfer.too.large.player.inventory=Rezept ist zu groß für 2x2
-jei.tooltip.error.crash=Tooltip-Fehler, siehe Log
+jei.tooltip.error.crash=Fehler des Tooltips, siehe Protokoll(Log)
 
 # Error Messages
 jei.chat.error.command.too.long=JEI muss auf dem Server installiert sein, damit dieser Befehl funktioniert, da er zu lang für Minecraft ist.
 jei.chat.error.no.cheat.permission.1=Dir ist es nicht erlaubt, JEI's Cheat-Modus zu benutzen.
-jei.chat.error.no.cheat.permission.2=Es ist allen erlaubt, die /give verwenden können oder im Kreativmodus sind.
+jei.chat.error.no.cheat.permission.2=Es ist allen erlaubt, die den /give Befehl verwenden können oder im Kreativmodus sind.
 
 # Key Bindings
-key.jei.toggleOverlay=Item-Liste (de-)aktivieren
+key.jei.toggleOverlay=Die Items-Liste (de-)aktivieren
```

Comment:
This is a term for the keybind menu, it should stay concise. I'd remove this change.
```suggestion
key.jei.toggleOverlay=Item-Liste (de-)aktivieren
```